lasagne/layers/special.py

```diff
@@ -6,12 +6,13 @@
 from ..utils import as_tuple
 from .base import Layer, MergeLayer
 
-
```

Comment:
This still needs to be fixed! We don't want random whitespace changes (at the best they make commit diffs longer than needed, and at the worst they cause merge conflicts with other PRs). Sorry for the hassle.